Loader/Config/Plugins/Server-Example Plugin.lua

```diff
@@ -36,7 +36,7 @@ return function(Vargs)
 		AdminLevel = "Players";	    -- Admin level; If using settings.CustomRanks set this to the custom rank name (eg. "Baristas")
 		Function = function(plr,args)    -- Function to run for command
 			print("HELLO WORLD FROM AN EXAMPLE COMMAND :)")
-			print("Player supplied args[1] "..tostring(args[1]))
+			print(`Player supplied args[1] {tostring(args[1])}`)
```

Comment:
Based on what ccuser44 just said, we no longer need a `tostring()` here. To simply replace the concats and do nothing else doesn't bring the full benefit.

I disagree with ccuser44 here and I think that we should promote this method of string manipulation. The negatives aren't important; you're looking at a string.

Reply:
> I disagree with ccuser44 here and I think that we should promote this method of string manipulation. The negatives aren't important; you're looking at a string.

Why? Theres no reason to do so unless there are the benefits.

Reply:
> > I disagree with ccuser44 here and I think that we should promote this method of string manipulation. The negatives aren't important; you're looking at a string.
> 
> Why? Theres no reason to do so unless there are the benefits.

The code looks more concise.

Reply:
> > > I disagree with ccuser44 here and I think that we should promote this method of string manipulation. The negatives aren't important; you're looking at a string.
> > 
> > 
> > Why? Theres no reason to do so unless there are the benefits.
> 
> The code looks more concise.

Not really, expecially when it doesn't colorise properly. Also it's -1 character so I would not really define that as more concise

Reply:
@Expertcoderz @LiveFireExercise @EasternBloxxer @LolloDev5123 @L8X whats your opinions?